pkg/generate/app/app.go

```diff
@@ -446,7 +462,18 @@ func (r *DeploymentConfigRef) DeploymentConfig() (*deployapi.DeploymentConfig, e
 		triggers = append(triggers, containerTriggers...)
 		template.Containers = append(template.Containers, *c)
 	}
-	// TODO: populate volumes
+
+	// Create EmptyDir volumes for all container volume mounts
+	for _, c := range template.Containers {
+		for _, v := range c.VolumeMounts {
+			template.Volumes = append(template.Volumes, kapi.Volume{
+				Name: v.Name,
+				VolumeSource: kapi.VolumeSource{
+					EmptyDir: &kapi.EmptyDirVolumeSource{Medium: ""},
```

Comment:
Use this [contant](https://github.com/GoogleCloudPlatform/kubernetes/blob/629237bbb77214256e41bbc0f998bb642058b6c7/pkg/api/types.go#L397) here